ai_worker/tests/src/test_schemas.py

```
@@ -0,0 +1,165 @@
+"""
+Test suite for data schemas
+"""
+
+import pytest
+from datetime import datetime
+from src.storage.schemas import EvidenceFile, EvidenceChunk, generate_file_id, generate_chunk_id
```

Pylint (4.0.3)

[error] No name 'storage' in module 'src'

(E0611)